doc/rest.md

```diff
@@ -0,0 +1,173 @@
+# REST
+
+When running and connected to a DotBot gateway, the `dotbot-controller`
+application provides a REST server to get information and control connected
+DotBots.
+
+The REST API is documented in the running `dotbot-controller` application itself
+at [http://localhost:8000/api](http://localhost:8000/api). This page also allows
+to play with the API directly from the browser.
+
+```{image} _static/images/pydotbot-ui-openapi.png
+:alt: Open API UI
+:class: bg-primary
+:width: 700px
+:align: center
+```
+
+## Prerequisites
+
+Make sure you already followed the [getting started](getting_started) page and
+have a functional setup with `dotbot-controller` running and connected to a
+nRF DK gateway.
```

Comment:
This is fine, but I would reverse the two phrases. It's stylistic, though, not necessary

Reply:
Let's keep it like this